.gitignore

```diff
@@ -1,7 +1,3 @@
 /node_modules/
 /typings/
-/jspm_packages/
-/dist/
-npm-debug.log
-.sw*
-.idea
```

Comment:
Okey, I think it is better to teach the users to have a clean environment but after all, not a big deal :)